lib/galaxy/datatypes/binary.py

```diff
@@ -1120,6 +1120,40 @@ def sniff( self, filename ):
 Binary.register_sniffable_binary_format("xlsx", "xlsx", Xlsx)
 
 
+class Xls( Binary ):
+    """Class describing an Excel (xls) file"""
+    file_ext = "xls"
+
+    def sniff( self, filename ):
+        ret = os.popen("file -i " + filename)
+        mimeType = ret.read().rstrip()
```

Comment:
Please use the [subprocess module](https://docs.python.org/2/library/subprocess.html) instead, e.g. `mimeType = subprocess.check_output("file -i %s" % filename).rstrip()`